tests/integration/tier0/latest-kernel-check/main.fmf

```diff
@@ -0,0 +1,27 @@
+summary: Verify that failed repoquery call does not cause latest kernel check call.
+
+tier: 0
+
+adjust+:
+    enabled: false
+    # TODO EUS Bump disabled version
+    when: >
+        distro != centos and
```

Comment:
`distro != centos` means any `centos` distribution?

Reply:
It does mean all `centos`